projects/components/src/filter-tabs/filter-tabs.component.scss

```diff
@@ -0,0 +1,46 @@
+@import 'layout';
+@import 'font';
+@import 'color-palette';
+
+.filter-tabs {
+  @include flex-layout(row);
+  @include body-1-medium($gray-5);
+  position: relative;
+  border: 1px solid $color-border;
+  border-radius: 6px;
+  background: $gray-1;
+  height: 36px;
+
+  .container {
+    display: flex;
+  }
+
+  .divider {
+    width: 1px;
+    margin: 6px 1px;
+    background: $gray-2;
+
+    &.adjacent-active {
+      background: $gray-1;
+    }
+  }
+
+  .active {
+    @include center-contents();
+    @include filter-box-shadow();
+    height: calc(100% - 4px); // Subtract 4px for margin
+    margin: 2px 0;
+    position: absolute;
+    width: 0; // Modified in html/ts
```

Comment:
This is a simple tab, it does not seem from an outside look like a situation where we need to use absolute, measure, set explicit widths etc. What's preventing this from being done using more simple layout?

Reply:
This is all to support the animations. The animation requires the movement of an element. But we only want the white background to move back and forth (left position) and adjust its width to match the parent tab.